src/test/index_management.rs

```diff
@@ -1,14 +1,35 @@
+#[path = "index_management/search_index.rs"]
+mod search_index_skip_ci;
+
 use futures::stream::TryStreamExt;
 
 use crate::{
     bson::doc,
     error::ErrorKind,
     options::{CommitQuorum, IndexOptions},
-    test::log_uncaptured,
+    test::{log_uncaptured, spec::unified_runner::run_unified_tests},
     Client,
     IndexModel,
 };
 
+#[tokio::test]
+async fn run_unified() {
+    let client = Client::for_test().await;
+
+    let mut skipped_files = Vec::new();
+    let mut skipped_tests = Vec::new();
+    // TODO DRIVERS-2794: unskip these tests
```

Comment:
I think changing around the test names changed the order in which the index management tests are being run such that we now hit the issue described in DRIVERS-2794